wrappers/hnygrpc/grpc_test.go

```diff
@@ -78,9 +78,11 @@ func TestUnaryInterceptor(t *testing.T) {
 	beeline.Init(beeline.Config{Client: client})
 
 	md := metadata.New(map[string]string{
-		"content-type": "application/grpc",
-		":authority":   "api.honeycomb.io:443",
-		"user-agent":   "testing-is-fun",
+		"content-type":      "application/grpc",
+		":authority":        "api.honeycomb.io:443",
```

Comment:
Same q re above and leading :

Reply:
Same answer: the leading colon is intentional.